.github/workflows/release-please.yml

```diff
@@ -27,6 +27,11 @@ jobs:
           check-latest: true
           registry-url: 'https://registry.npmjs.org'
         if: ${{ steps.release.outputs.release_created }}
+      - name: Setup Deno
```

Comment:
Adding this so that the test suite that runs as part of `npm publish` doesn't have to download the binary.